Tests/FunctionalTests/ResourcesTests.swift

```diff
@@ -88,4 +88,14 @@ class ResourcesTests: XCTestCase {
             }
         }
     }
+
+    func testFoundationlessClient() throws {
+        try fixture(name: "Resources/FoundationlessClient") throws {
+            #if os(Linux) && swift(>=5.7)
+            let pkgPath = fixturePath.appending(components: "AppPkg")
+            let buildPath = pkgPath.appending(components: ".build", UserToolchain.default.triple.platformBuildPathComponent(), "debug")
+            XCTAssertBuildFails(pkgPath)
```

Comment:
should we test on the failure reason also?

Reply:
For sure, we should; but it was late into the night and I wanted to have something up for CI testing to know if this approach is viable at all.

Pending further discussion also is the version gating both of the feature and of the test (including package manifest and test case here)—here both are lowered to 5.7 and up for ease of local and CI testing.